_posts/2021-06-23-#22261.md

```diff
@@ -83,9 +83,226 @@ commit:
 
 8. Bonus: Why does the unbroadcast set contain txids instead of wtxids?
 
-<!-- TODO: After meeting, uncomment and add meeting log between the irc tags
 ## Meeting Log
 
 {% irc %}
+
+17:00 <glozow> #startmeeting
+17:00 <LarryRuane> hi
+17:01 <schmidty> howdy
+17:01 <svav> hi
+17:01 <dopedsilicon> Hey
+17:01 <michaelfolkson> hola
+17:01 <Azorcode> Hi 
+17:01 <glozow> hi everyone! welcome to PR Review Club :) we're looking at Two small fixes to node broadcast logic today: https://bitcoincore.reviews/22261
+17:01 <lightlike> hi
+17:01 <glozow> did anyone get a chance to look at the notes or review the PR?
+17:02 <svav> Looked at the notes
+17:02 <michaelfolkson> Yup, maybe can't answer all the questions though
+17:03 <michaelfolkson> Took a while to get my head around why the fixes were needed
+17:03 <glozow> okie dokie
+17:03 → zakinator joined (~zakinator@c-73-152-14-173.hsd1.va.comcast.net)
+17:03 <glozow> so this PR is making changes to the `BroadcastTransaction()` function - can anyone tell me what this function does?
+17:03 → ru joined (~ru@122.174.144.66)
+17:03 <glozow> we're here: https://github.com/bitcoin/bitcoin/blob/7317e14a44c6efc545e6fb9bcedee7174e93a8fa/src/node/transaction.cpp#L29
```

Comment:
The commit hashes can be shortened to 7 or 8 characters, if you think it's more readable in the meeting log.I often shortened them.